code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -12,7 +12,7 @@
 
 /obj/item/gun/energy/e_gun/mini
```

Comment:
Can we give this to sec rather then normal disablers? They are rather under used